BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickRecyclerAdapter.java

```diff
@@ -70,11 +71,13 @@ public void safeNotifyDataSetChanged() {
      * @param position Position of the item that has changed
      * @param payload Optional parameter, use null to identify a "full" update
      */
-    public void safeNotifyItemChanged(final int position, final Object payload) {
-        if (recyclerView.isComputingLayout()) {
-            handler.post(new NotifyItemChangedWithPayloadRunnable(this, position, payload));
-        } else {
-            notifyItemChanged(position, payload);
+    public void safeNotifyItemChanged(final int position, @Nullable final Object payload) {
+        if (position >= 0) {
```

Comment:
**safety check**

Reply:
should this log a warning if position is < 0 to tell users they did something wrong?  Or if the library is broken?
Ditto the rest of these new safety checks?

Reply:
Hey @kenyee I added in logging of warnings for the safety checks.